### PR TITLE
Merge develop into master - v2.0.1

### PR DESCRIPTION
**Pull request by RHammond2**
@jordanperr I don't think I have the proper access to merge this into the master branch. Would you mind updating privileges or merging when you get a chance?

### CODE REVIEW

File: operational_analysis/types/asset.py
Comment (RHammond2) [reply operational_analysis/types/asset.py:135]: Exactly correct!

File: operational_analysis/types/asset.py
Comment (RHammond2) [reply operational_analysis/types/asset.py:6]: I don't believe that GDAL is used for pyrpoj or shapely, so this should be ok. In any case in `AssetData.parse_geometry()` there is a coordinate reference system conversion and that has to be handled with some geographic library. If the idea is to remove that functionality altogether, then I think we could simply use `geopy` for their geodesic distance function for "correct" turbine distances, or we could do a simple haversine distance implementation.

File: operational_analysis/types/asset.py
Comment (ejsimley) [inline operational_analysis/types/asset.py:20]: @RHammond2 great that you found a good alternative to geopandas. I added this note because of the earlier temporary fix where we were still using geopandas but not making it a requirement. So I would remove these lines since they aren't relevant anymore.

File: readme.md
Comment (ejsimley) [inline readme.md:36]: Similarly, this note about the Anaconda requirement for Windows and the GDAL package isn't relevant anymore, so I believe should be removed.

File: operational_analysis/types/asset.py
Comment (jordanperr) [reply operational_analysis/types/asset.py:6]: Eric and I spoke, sounds good!

File: CHANGELOG.md
Comment (jordanperr) [inline CHANGELOG.md:7]: Please add "Moved and renamed tests and updated the documentation accordingly."

**Review by jordanperr — APPROVED** [review]
Pending the small changes already mentioned here, I think it's good to go. Please wait for @ejsimley 's approval before merging.

File: operational_analysis/types/asset.py
Comment (RHammond2) [reply operational_analysis/types/asset.py:20]: Good catch! I'll update the doctsring.

**Review by ejsimley — APPROVED** [review]
This looks good to me now. Feel free to merge! Only thing I would say is that Rob can add his name to the contributors list in the readme.

File: CHANGELOG.md
Comment (RHammond2) [reply CHANGELOG.md]: Done.

**Comment by RHammond2** [comment]
@ejsimley my name was added in a different PR, but I've just gone and added here as well. Should I merge or wait for other changes to be merged into develop?

**Comment by ejsimley** [comment]
@RHammond2 I suggest merging this in now to take care of the dependency issues some users are having.

**Pull request merged**
